src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -2437,11 +2438,11 @@ def create_vmss(cmd, vmss_name, resource_group_name, image=None,
     master_template = ArmTemplateBuilder()
 
     uniform_str = 'Uniform'
-    flexible_str = 'Flexible'
-    if orchestration_mode.lower() == uniform_str.lower():
+    # flexible_str = 'Flexible'
+    if orchestration_mode:
```

Comment:
Default is uniform.

Reply:
I see. Just add here in case there is change to remove default

Reply:
No matter `orchestration_mode` is `Uniform` or `Flexible`, `if orchestration_mode` is always evaluated to `True`.